src/main/java/com/jmeter/websocket/plugin/configurations/WebsocketSessionsManager.java

```diff
@@ -0,0 +1,80 @@
+package com.jmeter.websocket.plugin.configurations;
+
+import com.google.common.base.Supplier;
+import com.jmeter.websocket.plugin.endpoint.WebsocketClient;
+import com.jmeter.websocket.plugin.endpoint.comsumers.CsvFileWriter;
+import com.jmeter.websocket.plugin.endpoint.jetty.JettyWebsocketEndpoint;
+import org.apache.jmeter.config.ConfigTestElement;
+import org.apache.jmeter.testelement.TestStateListener;
+import org.apache.jorphan.logging.LoggingManager;
+import org.apache.log.Logger;
+
+import java.nio.file.Paths;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static com.google.common.base.Suppliers.memoize;
+import static com.jmeter.websocket.plugin.endpoint.comsumers.CsvFileWriter.csvFileWriterSupplier;
+
+public class WebsocketSessionsManager extends ConfigTestElement implements TestStateListener {
+
+    private static final String FILE = "websocket.data.output.file";
+    private static final Logger log = LoggingManager.getLoggerForClass();
```

Comment:
Yep, but it will require Jmeter classpath update. Possible improvement for the next pull request.